sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -980,9 +977,7 @@ def download_manifest(self, repository: str, tag_or_digest: str, **kwargs) -> Do
                 digest = response.http_response.headers['Docker-Content-Digest']
             if not _validate_digest(manifest_stream, digest):
                 raise ValueError("The requested digest does not match the digest of the received manifest.")
-        except Exception:
-            if repository is None or tag_or_digest is None:
```

Comment:
Remove the `None` check because the default error is clear enough. When `repository` or `tag_or_digest` is `None`, will get the same error: `ValueError: No value for given attribute`